docs/courses/hello-world/1.2-blink-and-read.md

```diff
@@ -17,20 +17,64 @@ Learn how to write MicroPython scripts and use a Raspberry Pico W microcontrolle
 
 ### Getting Started with the Pico (Optional)
 
-Read the [Getting Started with the Pico](https://projects.raspberrypi.org/en/projects/getting-started-with-the-pico) tutorial, some which overlaps with prior content. Note that the onboard led for the Pico W is accessed via `onboard_led = Pin("LED", Pin.OUT)` instead of `onboard_led = Pin(25, Pin.OUT)`, which is for the original Pico. For the "Use digital inputs and outputs", "Control LED brightness with PWM", and "Control an LED with an analogue input" sections, you can still *run* the code without the optional hardware (button, wire-lead LED, resistor, potentiometer, breadboard, M-M jumper leads), though you won't be able to observe the effect.
-
-### Onboard LED
-
-```{note}
-The onboard LED is distinct from the RGB NeoPixel LED that you will be using later in the module.
-```
+<details>
+<summary>📚 Key Concepts from Getting Started Guide</summary>
+
+1. **Basic Pico W Setup**
+   ```python
+   # Essential imports
+   from machine import Pin
+   from time import sleep
+   
+   # LED setup
+   led = Pin("LED", Pin.OUT)
+   ```
+
+2. **Common Pin Functions**
+   - `Pin.OUT`: Set pin as output
+   - `Pin.IN`: Set pin as input
+   - `Pin.PULL_UP`: Enable internal pull-up resistor
+   - `Pin.PULL_DOWN`: Enable internal pull-down resistor
+
+3. **Quick Reference**
+   ```python
+   # Turn LED on/off
+   led.value(1)  # On
+   led.value(0)  # Off
+   
+   # Toggle LED
+   led.toggle()
+   ```
+
+4. **Pin Mapping**
+   <img src="./images/pico-pinout.png" alt="Pico W Pinout" width=400>
+</details>
+
+<details>
+<summary>🔧 Common Setup Issues & Solutions</summary>
```

Comment:
With the revamp of 1.1 running the demo (something I'd like to have a lot of oversight on), I think this may become unnecessary.